setup.py

```diff
@@ -56,13 +56,13 @@ def get_version(file="flake8_isort.py"):
     test_suite='run_tests',
     zip_safe=False,
     install_requires=[
-        'flake8 >= 3.2.1',
-        'isort[pyproject] >= 4.3.5',
-        'testfixtures',
+        'flake8 >= 3.2.1, <4',
+        'isort[pyproject] >= 4.3.5, <5',
```

Comment:
I suggest that this has a motivation in the comment:
```suggestion
        'isort[pyproject] >= 4.3.5, <5',  # https://github.com/gforcada/flake8-isort/issues/88
```